cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/HttpRequestBuilderImpl.java

```diff
@@ -132,4 +136,17 @@ public HttpRequestResponse execute() throws IOException {
             }
         }
     }
+
+    private Timer createTimer() {
+        if (!CwmsHttpClientMetrics.isMetricsEnabled()) {
+            return new NoOpTimer();
+        }
+        Metrics metrics = CwmsHttpClientMetrics.createMetrics(Objects.toString(httpUrl.resolve(endpoint)));
+        Timer timer = metrics.createTimer();
+        Properties metricsProperties = new Properties();
```

Comment:
bug in metrics api that needs to be resolved, https://jira.hecdev.net/browse/CWMS-734